package.json

```diff
@@ -11,5 +11,16 @@
     "npm",
     "package",
     "framorim"
-  ]
+  ],
+  "scripts": {
+    "lint": "eslint --ext .js --quiet src"
+  },
+  "devDependencies": {
+    "eslint": "^6.5.1",
+    "eslint-config-airbnb": "^18.0.1",
+    "eslint-plugin-import": "^2.18.2"
+  },
+  "dependencies": {
+    "yarn": "^1.19.1"
```

Comment:
We are not planning to use yarn at this time. Can you remove this from the dependencies?

Reply:
Updated.